openmc/mgxs/mgxs.py

```diff
@@ -510,27 +510,27 @@ def _create_tallies(self, scores, all_filters, keys, estimator):
         # Create each Tally needed to compute the multi group cross section
         for score, key, filters in zip(scores, keys, all_filters):
             self.tallies[key] = openmc.Tally(name=self.name)
-            self.tallies[key].add_score(score)
+            self.tallies[key].scores.append(score)
```

Comment:
Although I see why this works, I do believe that `score` will be a single `str`, and `filters` will be a list of filters, so the following should work:

```python
self.tallies[key].scores = [score]
self.tallies[key].filters = filters
```